packages/directlinespeech/src/createAdapters.js

```diff
@@ -79,7 +79,7 @@ export default async function create({
     const { privSource: source } = audioConfig;
 
     source.createAudioContext = () => {
-      if (!!source.privContext) {
+      if (!source.privContext) {
```

Comment:
This is to avoid linting error `no extra bool cast`

Reply:
But this also reverted the logic, right?